features/proposal/components/VoteCollapsableList/VoteCollapsableList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useMemo } from 'react';
+import { VoterDetail } from 'features/types';
+import groupBy from 'lodash/groupBy';
+
+import { NoResultsView } from 'astro_2.0/components/NoResultsView';
+import { Collapsable } from 'components/collapsable/Collapsable';
+import { VoteCollapsableHeader } from 'features/proposal/components/VoteCollapsableHeader';
+import { VotersList } from 'features/proposal/components/VotersList';
+
+import styles from './VoteCollapsableList.module.scss';
+
+interface VoteCollapsableListProps {
+  data: VoterDetail[];
+}
+
+export const VoteCollapsableList: FC<VoteCollapsableListProps> = ({ data }) => {
+  const votesGroups: { [key: string]: VoterDetail[] } = useMemo(
+    () => groupBy(data, 'groups'),
+    [data]
+  );
+
+  if (!data?.length) {
```

Comment:
we can use lodash `isEmpty` which is more laconic

Reply:
ok, I will fix